src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -80,6 +80,11 @@ public new CompressionLevelWithAuto RequestBodyCompressionLevel
             }
         }
 
+        /// <summary>
+        /// Whether the SDK should be tracking the session automatically.
+        /// </summary>
+        public bool EnableAutoSessionTracking { get; set; } = false;
```

Comment:
```suggestion
        /// <summary>
        /// Whether the SDK should be tracking the session automatically.
        /// </summary>
        /// <remakrs>
        /// This will come from the .NET SDK main options and can be removed in the future.
        /// </remarks>
        public bool EnableAutoSessionTracking { get; set; }
```